lib/index.ts

```diff
@@ -276,10 +132,14 @@ module.exports = function (stripe) {
       if (plan) {
         if (sub.plan.id !== plan) {
           debug('update subscription plan:', plan);
-          await stripe.subscriptions.update(sub.id, {
+          let trial_end;
+          if (trialDays) {
```

Comment:
This fixes a bug where `trialDays` wouldn't do anything if the subscription already existed. `trial_period_days` is not a valid attribute when updating a subscription but a `trial_end` date is. This will make the `trialDays` count form today.